app/hotels/src/allHotels/AllHotelsSearchList.js

```diff
@@ -22,10 +22,10 @@ type HotelType = {|
 
 export class AllHotelsSearchList extends React.Component<Props> {
   componentDidMount() {
-    const hotelId = this.props.data[0]?.hotelId;
```

Comment:
What's the reason behind this refactor?

Reply:
Because flow type definition of `data` was wrong